document/concepts/overview.rst

```diff
@@ -1,6 +1,91 @@
 Overview
 --------
 
-.. todo::
+WebAssembly encodes a low-level, assembly-like programming language.
+This language is structured around the following main concepts.
 
-   Describe
+* **Values**.
+  WebAssembly provides only four basic *value types*.
+  These are integers and :ref:`IEEE-754 floating point <http://ieeexplore.ieee.org/document/4610935/>` numbers,
+  each in 32 and 64 bit width.
+  32 bit integers also function as Booleans and as memory addresses.
+  The usual basic operations on these types are available,
+  including the full matrix of conversions between them.
+  There is no distinction between signed and unsigned integer types.
+  Instead, different signedness is handled by different operations,
```

Comment:
Instead, the signedness interpretation of integer bit patterns is specified only by those operations where it makes a difference in the outcome. Signed numbers are interpreted as 2's complement.